extensions/github1s/src/views/pull-list-view.ts

```diff
@@ -6,49 +6,126 @@
 import * as vscode from 'vscode';
 import { relativeTimeTo } from '@/helpers/date';
 import repository, { RepositoryPull } from '@/repository';
+import { GitHub1sSourceControlDecorationProvider } from '@/providers/sourceControlDecorationProvider';
+import {
+	getChangedFileCommand,
+	getPullChangeFiles,
+} from '@/source-control/changes';
 
-const getPullTreeItemDescription = (pull: RepositoryPull) => {
+enum PullStatus {
+	OPEN = 'open',
+	CLOSED = 'closed',
+	MERGED = 'merged',
+}
+
+const getPullStatus = (pull: RepositoryPull): PullStatus => {
 	// current pull request is open
 	if (pull.state === 'open') {
-		return `opened ${relativeTimeTo(pull.created_at)} by ${pull.user.login}`;
+		return PullStatus.OPEN;
 	}
 
 	// current pull request is merged
 	if (pull.state === 'closed' && pull.merged_at) {
+		return PullStatus.MERGED;
```

Comment:
Should we use `PullState.OPEN`, `PullState.closed` to relace `'open'`, `'closed'` here?

Reply:
+1, I will replace it, thanks!